src/rule-must-colocate-fragment-spreads.js

```diff
@@ -153,15 +167,33 @@ function rule(context) {
 
     ImportDeclaration(node) {
       if (node.importKind === 'value') {
-        foundImportedModules.push(utils.getModuleName(node.source.value));
+        node.specifiers.forEach(specifier => {
+          if (
+            allowNamedImports &&
+            specifier.imported &&
+            isFirstLetterUppercase(specifier.imported.name)
+          ) {
+            foundImportedModules.push({
+              type: 'namedImport',
+              value: specifier.imported.name
```

Comment:
The name being imported is independent of the module that the export defined in - but the rest of the logic is based on the `value` indicating the module where the import is defined (and therefore the name of the fragment that is defined in it).

e.g. `./moda.ts` can define `iUseTheFragment` which takes `modaFragment$key` as an argument. But this code seems to assume that because the imported name is `iUseTheFragment`, that the relevant fragment is called `iUseTheFragment` - but actually it's called `modaFragment`.